stable/graylog/values.yaml

```diff
@@ -125,6 +125,9 @@ graylog:
     port: 9000
 
     master:
+      ## enable Graylog master service
+      ##
+      enabled: true
```

Comment:
Can you give an example when master service need to be disabled?

Reply:
I wasn’t sure when to use the master or the web service. Ultimately I went with the web service so that it could be load balancer across the pods since graylog works perfectly fine from any instance. If the master goes down then the service is down + it doesn’t reallocate the label until the old master comes up. So it seemed more resilient to always use the web over the master, unless I’m missing the use cases for either service?

Reply:
There are some use cases which we will use Graylog API to run additional features e.g. indices management. The master service is required. And new pod can be labeled as master if master cannot be found.

Reply:
We discover the master via (pods on boot up)[https://github.com/helm/charts/blob/master/stable/graylog/templates/configmap.yaml#L143-L161], so the service does not play a role in that. Also if the master goes offline, is there an election process that adds a label to a coordinating node? I don't think there is so referencing the master service could lead to outages while it's down. 

Can you talk a bit more about your use case? I'm curious because we are trying to figure out which service to expose and the web service seems to hold all the same functionality as the master even when dealing with index management?

Reply:
In my production, I have several TB of logs and indices older than 3 months usually are closed. Sometime, we need to reopen old indices. If we reopen indices from Elasticsearch directly, the Graylog will check the state and close them again almost immediately. To reopen closed indices properly, we need to Graylog UI. But sometime we don't know which indices need to be opened so I run a script to send command to master node to open indices, do something and closed indices when it operation is completed.